<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>Chapter1 Literature Review | Risk Prediction of Primary Ovarian Insufficiency in Childhood Cancer Survivors</title>
  <meta name="description" content="Chapter1 Literature Review | Risk Prediction of Primary Ovarian Insufficiency in Childhood Cancer Survivors" />
  <meta name="generator" content="bookdown 0.23 and GitBook 2.6.7" />

  <meta property="og:title" content="Chapter1 Literature Review | Risk Prediction of Primary Ovarian Insufficiency in Childhood Cancer Survivors" />
  <meta property="og:type" content="book" />
  
  
  
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="Chapter1 Literature Review | Risk Prediction of Primary Ovarian Insufficiency in Childhood Cancer Survivors" />
  
  
  

<meta name="author" content="Lin Yu" />


<meta name="date" content="2021-09-05" />

  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="index.html"/>
<link rel="next" href="polygenic-risk-score-construction.html"/>
<script src="libs/header-attrs-2.6/header-attrs.js"></script>
<script src="libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />









<link href="libs/anchor-sections-1.0.1/anchor-sections.css" rel="stylesheet" />
<script src="libs/anchor-sections-1.0.1/anchor-sections.js"></script>



<style type="text/css">
/* Used with Pandoc 2.11+ new --citeproc when CSL is used */
div.csl-bib-body { }
div.csl-entry {
  clear: both;
}
.hanging div.csl-entry {
  margin-left:2em;
  text-indent:-2em;
}
div.csl-left-margin {
  min-width:2em;
  float:left;
}
div.csl-right-inline {
  margin-left:2em;
  padding-left:1em;
}
div.csl-indent {
  margin-left: 2em;
}
</style>

<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./">Lin Yu's Thesis</a></li>

<li class="divider"></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>Introduction</a></li>
<li class="chapter" data-level="1" data-path="Lit.html"><a href="Lit.html"><i class="fa fa-check"></i><b>1</b> Literature Review</a>
<ul>
<li class="chapter" data-level="1.1" data-path="Lit.html"><a href="Lit.html#background01"><i class="fa fa-check"></i><b>1.1</b> Background</a></li>
<li class="chapter" data-level="1.2" data-path="Lit.html"><a href="Lit.html#methodology-review"><i class="fa fa-check"></i><b>1.2</b> Methodology Review</a>
<ul>
<li class="chapter" data-level="1.2.1" data-path="Lit.html"><a href="Lit.html#polygenic-risk-score"><i class="fa fa-check"></i><b>1.2.1</b> Polygenic Risk Score</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="2" data-path="polygenic-risk-score-construction.html"><a href="polygenic-risk-score-construction.html"><i class="fa fa-check"></i><b>2</b> Polygenic Risk Score Construction</a></li>
<li class="chapter" data-level="3" data-path="risk-prediction-for-primary-ovarian-insufficiency-in-childhood-cancer-survivors.html"><a href="risk-prediction-for-primary-ovarian-insufficiency-in-childhood-cancer-survivors.html"><i class="fa fa-check"></i><b>3</b> Risk Prediction for Primary Ovarian Insufficiency in Childhood Cancer Survivors</a>
<ul>
<li class="chapter" data-level="3.1" data-path="risk-prediction-for-primary-ovarian-insufficiency-in-childhood-cancer-survivors.html"><a href="risk-prediction-for-primary-ovarian-insufficiency-in-childhood-cancer-survivors.html#introduction-1"><i class="fa fa-check"></i><b>3.1</b> Introduction</a></li>
<li class="chapter" data-level="3.2" data-path="risk-prediction-for-primary-ovarian-insufficiency-in-childhood-cancer-survivors.html"><a href="risk-prediction-for-primary-ovarian-insufficiency-in-childhood-cancer-survivors.html#methods"><i class="fa fa-check"></i><b>3.2</b> Methods</a></li>
<li class="chapter" data-level="3.3" data-path="risk-prediction-for-primary-ovarian-insufficiency-in-childhood-cancer-survivors.html"><a href="risk-prediction-for-primary-ovarian-insufficiency-in-childhood-cancer-survivors.html#results"><i class="fa fa-check"></i><b>3.3</b> Results</a></li>
<li class="chapter" data-level="3.4" data-path="risk-prediction-for-primary-ovarian-insufficiency-in-childhood-cancer-survivors.html"><a href="risk-prediction-for-primary-ovarian-insufficiency-in-childhood-cancer-survivors.html#discussion"><i class="fa fa-check"></i><b>3.4</b> Discussion</a></li>
<li class="chapter" data-level="3.5" data-path="risk-prediction-for-primary-ovarian-insufficiency-in-childhood-cancer-survivors.html"><a href="risk-prediction-for-primary-ovarian-insufficiency-in-childhood-cancer-survivors.html#reference"><i class="fa fa-check"></i><b>3.5</b> Reference</a></li>
</ul></li>
<li class="chapter" data-level="4" data-path="conclusion.html"><a href="conclusion.html"><i class="fa fa-check"></i><b>4</b> Conclusion</a></li>
<li class="chapter" data-level="5" data-path="final-words.html"><a href="final-words.html"><i class="fa fa-check"></i><b>5</b> Final Words</a></li>
<li class="chapter" data-level="" data-path="references.html"><a href="references.html"><i class="fa fa-check"></i>References</a></li>
<li class="divider"></li>
<li><a href="https://github.com/rstudio/bookdown" target="blank">Published with bookdown</a></li>

</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Risk Prediction of Primary Ovarian Insufficiency in Childhood Cancer Survivors</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="Lit" class="section level1" number="1">
<h1><span class="header-section-number">Chapter1</span> Literature Review</h1>
<div id="background01" class="section level2" number="1.1">
<h2><span class="header-section-number">1.1</span> Background</h2>
<p>It is estimated that approximately 90% of women experience menopause between the age of45 and 55 years [ref], with the median age at natural menopause occurring between 50 and 52 years of age[ref]. Primary ovarian insufficiency (POI) occurs if a woman experiences menopause naturally before age 40, and the prevalence of POI is about 1% in the general population.[ref]
However, among childhood cancer survivors, the continually growing five-year survival rate (exceeds 80% already) adds to an increasing prevalence of POI as a result of cancer therapies, which accounts for about 10% of cancer survivors.[ref]</p>
<p><strong><em>Impact of Primary Ovarian Insufficiency</em></strong></p>
<p>Individuals with POI are more likely to develop chronic diseases. Shah et al. have shown that patients have an increased probability of developing cardiovascular disease in the postmenopause years [ref]. Lower levels of estrogen following menopause also increase the risk of developing hypertension and ventricular remodeling.[ref] At the same time, menopause along with the chronic diseases also place a mental strain on both patients and their families. It has been shown that women with ovarian dysfunction were more likely to develop anxiety and depression.[ref]</p>
<p><strong><em>Fertility Preservation</em></strong></p>
<p>Another primary concern of POI is infertility, as fertility is irreversible after POI onset. Some fertility preservation options, such as oocyte and ovarian tissue cryopreservation, are available to preserve reproductive function [ref]. However, a study has suggested that cancer patients feel difficult to make decisions about fertility preservation [ref]. One reason is that these decisions are time-pressured. Many participants reported that they did not have enough time for decision-making before cancer treatment[ref]. Additionally, uncertainty makes it challenging to make decisions[ref]. For example, women have to trade-off the risk of developing POI after cancer treatments with no guarantee of favorable outcomes from the fertility preservation procedure [ref]. Some decision aid methods have been proposed [ref] , which mainly focused on improving patients’ knowledge about fertility preservation.</p>
<p><strong><em>Clinical Risk Factors and Risk Prediction Model</em></strong></p>
<p>Extensive studies have been undertaken to identify treatment-related risk factors associated with compromised reproductive function following cancer treatment[ref]. Chemotherapy agents, especially alkylating agents (such as busulfan, cyclophosphamide, lomustine, and procarbazine, etc.), can prevent cell division and growth by interacting with DNA and reduce the number of follicles for maturation and reproduction, increasing the risk for ovarian dysfunction [ref] . Also, radiation to the ovary, abdominal or pelvic sites can induce genomic damage in oocytes and the surrounding granulosa cells, leading to either a decreased or exhausted ovarian follicle pool depending on the extent of the damage [ref] .</p>
<p>Well-established clinical risk factors and cancer treatments mentioned before have been evaluated as risk factors to develop risk prediction models for compromised reproductive function in female cancer survivors. For example, Clark et al. has used clinical predictors such as cumulative alkylating drug dose [ref], radiation exposure to the ovary, abdomen and pelvis, age at cancer diagnosis, and hematopoietic stem-cell transplant receipt to build models to predict individual’s risk for developing menopause within five years following the time of cancer diagnosis. Zhe has investigated the association between clinical risk factors, their potential interactions on POI, and incorporated them into a predictive model for age-specific POI risk in female cancer survivors. [ref]</p>
<p><strong><em>Genetic Risk for Primary Ovarian Insufficiency</em></strong></p>
<p>Aside from clinical risk factors, genetic studies have shown that POI is a complex, heterogeneous disorder that is influenced by multiple genetic components.[ref] Genetic studies have shown that the genetic variations on the X chromosome contribute mostly to the etiology of POI. Meanwhile, increasing attention has focused on autosome single-gene variations known to regulate follicle development and maturation.[ref] For example, Rajkovic et al. discovered that the Newborn ovary homeobox gene (NOBOX) plays a critical role in early folliculogenesis [ref]. Deficiency in NOBOX disrupts early folliculogenesis and oocyte-specific gene expression, accelerating post-natal oocyte loss and abolishing the transition from primordial to growing follicles [ref]. Moreover, The mutations in follicle-stimulating hormone receptor genes, such as FSHR and LHCGR, were found to affect the development and maturation of follicles and oocytes [ref]. So far, more than sixty genetic variants have been identified, accounting for 10–13% of the variance of POI in the general population [ref]. A list of several genes that affect the development and maturation of POI is included in the appendix</p>
<p><strong><em>Genetic Variants and Genome-wide Association Study (GWAS)</em></strong></p>
<p>DNA sequence is formed from a chain of four nucleotide bases: A, C, G and T. All humans have near-identical DNA sequences across the estimated six billion-letter codes for their genome13. However, slight differences exist between individuals. If more than 1% of a population does not carry the same nucleotide at a specific position within the DNA sequence, then this variation can be classified as a single nucleotide polymorphism (SNP) or genetic variant14.</p>
<p>Many techniques were applied to discover the genetic variants for different phenotypes. A popular technique, Genome-wide Association Study (GWAS) has been widely conducted to study the genetics of phenotypes in recent decades11, which aims to find common variants12 . GWAS technique consists of screening and comparing genetic variants in patients with the disease of interest ( or phenotype) and healthy controls. In GWAS, the association between a genetic variant to the phenotype is observed when there is a statistically significant difference of the genetic variant between diseased patients and healthy controls.</p>
<p>GWAS studies have been conducted in several ancestries. As of 2019, 78% of the GWAS studies were performed in the European ancestry, 10% were conducted in Asian ancestry. For POI trait, there exist five GWAS studies to date, all five studies have limited sample size, with the largest sample size being around 1300. four of them were performed in the general population in either European or East Asian ancestries, and the remaining one was conducted in the childhood cancer survivors. a summary of the studies was provided in the appendix the table from cat log
more GWASs studies with a larger study sample size were available for age at natural menopause. 14 GWAS studies are found and the largest one is Day et al, with #69626 individuals of European ancestry.</p>
<p><strong><em>GWAS data for Age at Natural Menopause in the General Population</em></strong>
Ideally, the GWAS information of POI is the perfect data source for studying the association between common variants and POI. However, the existing GWAS data for POI in the general population was limited by the small total study population sample size of the GWAS studies. Instead, this study focused on the GWAS data of age at natural menopause in the general population, as it has been hypothesized that age at natural menopause and POI are possibly manifestations of the same underlying genetic susceptibility owing to the inheritance patterns observed. [ref] Studies also showed that POI and age at natural menopause share common genetic factors involved in DNA repair and maintenance [ref]. The following describes some main findings of the two most recent GWAS studies with a reasonable sample size for age at natural menopause in the general population.</p>
<p>A meta-analysis of GWAS conducted in the general population analyzed up to 69,360 women of European ancestry. The study showed that 1,208 out of a total of ~2.6 million genetic variants, reached the genome-wide significance threshold (P &lt; 5 × 10−8) for association with age at natural menopause. Among those significant genetic variants, 54 independent genetic variants at 44 genetic loci were associated with age at menopause, explaining 6% of the variance in age at natural menopause. The variance explained increased to 21% for the top 29,958 independent variants with an association P value less than 0.05.[ref]</p>
<p>European population PRS69 [TO BE FINISHED]</p>
<p><strong><em>GWAS data for Primary Ovarian Insufficiency in the Cancer Survivors</em></strong></p>
<p>The genetics of menopause in the cancer survivor population is less investigated. To date, only Brooke et al. have conducted a GWAS study in the cancer survivor population.[ref] The study conducted GWAS to identify genetic variants associated with clinically diagnosed POI in 799 female cancer survivors in the St. Jude Lifetime Cohort Study (SJLIFE). The analyses were adjusted for cyclophosphamide equivalent dose of alkylating agents and ovarian radiotherapy (RT) dose. The results showed that 20 out of 830 884 genotyped variants had a P value less than 10-5. Among them, 13 genetic variants (upstream of the Neuropeptide Receptor 2 gene) were seen in more than 50% of POI patients and were associated with POI prevalence. A haplotype formed by 4 of the 13 variants showed association with POI for patients exposed to ovarian RT, indicating interaction between genetics and radiation treatment. Replication was performed in 1624 survivors from the Childhood Cancer Survivor Study (CCSS), and the association was also observed between the haplotype and POI for patients exposed to ovarian radiotherapy (OR= 3.97, 95%CI=[1.67 to 9.41], P= .002).</p>
<p><strong><em>Potential of Polygenic Risk Score</em></strong></p>
<p>Although the increasing use of GWAS has led to the identification of many variants associated with POI, the effect size of genetic variants identified from GWAS are typically small and account for only a small fraction of association [ref], meaning that single variants have limited predictive power[ref]. Thus, this study evaluated the genetic risk for POI in the form of a polygenic risk score (PRS), or a score that combines the estimated effects of many disease-associated genetic variants reported in published GWAS data. PRSs have been proposed as a genetic risk prediction tool for a wide range of diseases [ref]; According to PGS catalog, an open database of published polygenic risk scores, 829 polygenic risk scores were built over 214 traits as of July 2021. A clinically-useful PRS would allow clinicians to identify individuals at elevated risk of disease, thus informing disease screening, therapeutic interventions, and life planning to prevent or delay the onset of disease.</p>
</div>
<div id="methodology-review" class="section level2" number="1.2">
<h2><span class="header-section-number">1.2</span> Methodology Review</h2>
<div id="polygenic-risk-score" class="section level3" number="1.2.1">
<h3><span class="header-section-number">1.2.1</span> Polygenic Risk Score</h3>
<p>A polygenic risk score or PRS (also termed a polygenic score or genetic risk score) estimates an individual’s genetic liability to a trait or disease.[ref] PRS is derived from genotype profile and relevant genome-wide association study (GWAS) data. PRS is a sum of genome-wide genotypes, weighted by corresponding genotype effect size estimates derived from GWAS summary statistics. The general steps for computing PRS include selecting base and target data, quality control of base and target data, finally, the construction of PRS.</p>
<p><strong><em>Base and Target Data</em></strong></p>
<p>Two key input data sets are required to characterize PRS based on the definition. The base data (GWAS studies), consisting of summary statistics of the association between genetic variants and phenotypes, are needed to characterize PRS. Target data, usually new samples independent of the base samples and with genotype and phenotype data for each individual, are used to perform the PRS analysis.</p>
<p>The base data (i.e., summary statistics of GWAS) are usually publicly available. the following summary statistic information is needed for the PRS construction:</p>
<p><strong><em>Selection of GWAS</em></strong></p>
<p>Several aspects were considered when it came to the selection of GWAS studies, including the sample size of the GWAS, the ancestry of the samples, the quality control process, and the phenotypes being studied in GWAS.</p>
<p><strong>Sample Size of GWAS</strong>: Given that the PRS is built based on the summary statistics from GWAS studies, the GWAS studies should have a reasonable sample size so that it is powerful enough to detect the association between genetic variants and phenotype and ultimately increase the predictive ability of the PRS. GWAS studies with a sample size larger than 10,000 are recommended.</p>
<p><strong>Ancestry of the Sample</strong>: The genetic profiles of different ancestry groups are significantly different, as the allele frequencies and the correlation between genetic variants vary among different ancestry groups. The estimated effect size of a given genetic variant might be different from population to population. Therefore, the PRS generated from GWAS summary statistics using target samples is usually limited to a specific population. For this reason, the ancestry of the base and target samples should be the same to ensure an accurate estimation.</p>
<p><strong>Phenotype Being Studied</strong>: The phenotype studied in the base data should be the same as the phenotype of interest in the PRS construction. However, some phenotypes might share similar genetic architecture. Thus GWAS studies for phenotypes related to the phenotype of interest could be a substitute when GWAS studies for phenotypes of interest are not available.</p>

</div>
</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="index.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="polygenic-risk-score-construction.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"whatsapp": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": null,
"text": null
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": ["Lin_Thesis_Book.pdf", "Lin_Thesis_Book.epub"],
"search": {
"engine": "lunr",
"options": null
},
"toc": {
"collapse": "subsection"
}
});
});
</script>

</body>

</html>
