<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>Chapter1 Literature Review | Risk Prediction of Primary Ovarian Insufficiency in Childhood Cancer Survivors</title>
  <meta name="description" content="Chapter1 Literature Review | Risk Prediction of Primary Ovarian Insufficiency in Childhood Cancer Survivors" />
  <meta name="generator" content="bookdown 0.23 and GitBook 2.6.7" />

  <meta property="og:title" content="Chapter1 Literature Review | Risk Prediction of Primary Ovarian Insufficiency in Childhood Cancer Survivors" />
  <meta property="og:type" content="book" />
  
  
  
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="Chapter1 Literature Review | Risk Prediction of Primary Ovarian Insufficiency in Childhood Cancer Survivors" />
  
  
  

<meta name="author" content="Lin Yu" />


<meta name="date" content="2021-09-06" />

  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="index.html"/>
<link rel="next" href="polygenic-risk-score-construction.html"/>
<script src="libs/header-attrs-2.6/header-attrs.js"></script>
<script src="libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />









<link href="libs/anchor-sections-1.0.1/anchor-sections.css" rel="stylesheet" />
<script src="libs/anchor-sections-1.0.1/anchor-sections.js"></script>



<style type="text/css">
/* Used with Pandoc 2.11+ new --citeproc when CSL is used */
div.csl-bib-body { }
div.csl-entry {
  clear: both;
}
.hanging div.csl-entry {
  margin-left:2em;
  text-indent:-2em;
}
div.csl-left-margin {
  min-width:2em;
  float:left;
}
div.csl-right-inline {
  margin-left:2em;
  padding-left:1em;
}
div.csl-indent {
  margin-left: 2em;
}
</style>

<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./">Lin Yu's Thesis</a></li>

<li class="divider"></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>Introduction</a></li>
<li class="chapter" data-level="1" data-path="literature-review.html"><a href="literature-review.html"><i class="fa fa-check"></i><b>1</b> <strong>Literature Review</strong></a>
<ul>
<li class="chapter" data-level="1.1" data-path="literature-review.html"><a href="literature-review.html#methodology-review"><i class="fa fa-check"></i><b>1.1</b> <strong>Methodology Review</strong></a>
<ul>
<li class="chapter" data-level="1.1.1" data-path="literature-review.html"><a href="literature-review.html#polygenic-risk-score"><i class="fa fa-check"></i><b>1.1.1</b> Polygenic Risk Score</a></li>
<li class="chapter" data-level="1.1.2" data-path="literature-review.html"><a href="literature-review.html#risk-prediction"><i class="fa fa-check"></i><b>1.1.2</b> Risk Prediction</a></li>
<li class="chapter" data-level="1.1.3" data-path="literature-review.html"><a href="literature-review.html#evaluation-of-model-performance"><i class="fa fa-check"></i><b>1.1.3</b> Evaluation of Model Performance</a></li>
</ul></li>
<li class="chapter" data-level="1.2" data-path="literature-review.html"><a href="literature-review.html#the-ccss-original-cohort"><i class="fa fa-check"></i><b>1.2</b> <strong>The CCSS Original Cohort</strong></a></li>
<li class="chapter" data-level="1.3" data-path="literature-review.html"><a href="literature-review.html#references"><i class="fa fa-check"></i><b>1.3</b> <strong>References</strong></a></li>
</ul></li>
<li class="chapter" data-level="2" data-path="polygenic-risk-score-construction.html"><a href="polygenic-risk-score-construction.html"><i class="fa fa-check"></i><b>2</b> Polygenic Risk Score Construction</a></li>
<li class="chapter" data-level="3" data-path="risk-prediction-for-primary-ovarian-insufficiency-in-childhood-cancer-survivors.html"><a href="risk-prediction-for-primary-ovarian-insufficiency-in-childhood-cancer-survivors.html"><i class="fa fa-check"></i><b>3</b> Risk Prediction for Primary Ovarian Insufficiency in Childhood Cancer Survivors</a>
<ul>
<li class="chapter" data-level="3.1" data-path="risk-prediction-for-primary-ovarian-insufficiency-in-childhood-cancer-survivors.html"><a href="risk-prediction-for-primary-ovarian-insufficiency-in-childhood-cancer-survivors.html#introduction-1"><i class="fa fa-check"></i><b>3.1</b> Introduction</a></li>
<li class="chapter" data-level="3.2" data-path="risk-prediction-for-primary-ovarian-insufficiency-in-childhood-cancer-survivors.html"><a href="risk-prediction-for-primary-ovarian-insufficiency-in-childhood-cancer-survivors.html#methods"><i class="fa fa-check"></i><b>3.2</b> Methods</a></li>
<li class="chapter" data-level="3.3" data-path="risk-prediction-for-primary-ovarian-insufficiency-in-childhood-cancer-survivors.html"><a href="risk-prediction-for-primary-ovarian-insufficiency-in-childhood-cancer-survivors.html#results"><i class="fa fa-check"></i><b>3.3</b> Results</a></li>
<li class="chapter" data-level="3.4" data-path="risk-prediction-for-primary-ovarian-insufficiency-in-childhood-cancer-survivors.html"><a href="risk-prediction-for-primary-ovarian-insufficiency-in-childhood-cancer-survivors.html#discussion"><i class="fa fa-check"></i><b>3.4</b> Discussion</a></li>
<li class="chapter" data-level="3.5" data-path="risk-prediction-for-primary-ovarian-insufficiency-in-childhood-cancer-survivors.html"><a href="risk-prediction-for-primary-ovarian-insufficiency-in-childhood-cancer-survivors.html#reference"><i class="fa fa-check"></i><b>3.5</b> Reference</a></li>
</ul></li>
<li class="chapter" data-level="4" data-path="conclusion.html"><a href="conclusion.html"><i class="fa fa-check"></i><b>4</b> Conclusion</a></li>
<li class="chapter" data-level="5" data-path="final-words.html"><a href="final-words.html"><i class="fa fa-check"></i><b>5</b> Final Words</a></li>
<li class="chapter" data-level="" data-path="references-1.html"><a href="references-1.html"><i class="fa fa-check"></i>References</a></li>
<li class="divider"></li>
<li><a href="https://github.com/rstudio/bookdown" target="blank">Published with bookdown</a></li>

</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Risk Prediction of Primary Ovarian Insufficiency in Childhood Cancer Survivors</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="literature-review" class="section level1" number="1">
<h1><span class="header-section-number">Chapter1</span> <strong>Literature Review</strong></h1>
<p>Background</p>
<p>It is estimated that approximately 90% of women experience menopause between the age of 45 and 55 years<a href="https://www.zotero.org/google-docs/?qizsug">[10]</a>, with the median age at natural menopause occurring between 50 and 52 years of age<a href="https://www.zotero.org/google-docs/?Eq5IY8">[11]</a>. Primary ovarian insufficiency (POI) occurs if a woman experiences menopause naturally before age 40, and the prevalence of POI is about 1%  in the general population<a href="https://www.zotero.org/google-docs/?Hk9bpr">[12]</a>.</p>
<p>However, among childhood cancer survivors, the continually growing five-year survival rate (exceeds 80% already) adds to an increasing prevalence of POI as a result of cancer therapies, which accounts for about 10% of cancer survivors<a href="https://www.zotero.org/google-docs/?jjdj0Q">[13]</a>.</p>
<p>Impact of Primary Ovarian Insufficiency  </p>
<p>Individuals with POI are more likely to develop chronic diseases. Shah et al. have shown that patients have an increased probability of developing cardiovascular disease in the postmenopause years<a href="https://www.zotero.org/google-docs/?VHlbNU">[14]</a>. Lower estrogen levels following menopause also increase the risk of developing hypertension and ventricular remodeling<a href="https://www.zotero.org/google-docs/?5UIooz">[15]</a>. At the same time, menopause and chronic diseases also place a mental strain on both patients and their families. It has been shown that women with ovarian dysfunction were more likely to develop anxiety and depression.[ref]</p>
<p>Fertility Preservation</p>
<p>Another primary concern of POI is infertility, as fertility is irreversible after POI onset. Some fertility preservation options, such as oocyte and ovarian tissue cryopreservation, are available to preserve reproductive function [ref]. However, a study has suggested that cancer patients feel challenging to make decisions about fertility preservation [ref]. One reason is that these decisions are time-pressured. Many participants reported that they did not have enough time for decision-making before cancer treatment[ref]. Additionally, uncertainty makes it challenging to make decisions[ref]. For example, women have to trade-off the risk of developing POI after cancer treatments with no guarantee of favorable outcomes from the fertility preservation procedure [ref]. Some decision aid methods have been proposed [ref] , which mainly focused on improving patients’ knowledge about fertility preservation. </p>
<p>Clinical Risk Factors and Risk Prediction Model </p>
<p>Extensive studies have been undertaken to identify treatment-related risk factors associated with compromised reproductive function following cancer treatment[ref]. Chemotherapy agents, especially alkylating agents (such as busulfan, cyclophosphamide, lomustine, and procarbazine, etc.), can prevent cell division and growth by interacting with DNA and reduce the number of follicles for maturation and reproduction, increasing the risk for ovarian dysfunction [ref] . Also, radiation to the ovary, abdominal or pelvic sites can induce genomic damage in oocytes and the surrounding granulosa cells, leading to either a decreased or exhausted ovarian follicle pool depending on the extent of the damage [ref] .</p>
<p>Well-established clinical risk factors and cancer treatments mentioned before have been evaluated as risk factors to develop risk prediction models for compromised reproductive function in female cancer survivors. For example, Clark et al. has used clinical predictors such as cumulative alkylating drug dose [ref], radiation exposure to the ovary, abdomen and pelvis, age at cancer diagnosis, and hematopoietic stem-cell transplant receipt to build models to predict individual’s risk for developing menopause within five years following the time of cancer diagnosis. Zhe has investigated the association between clinical risk factors and their potential interactions on POI and incorporated them into a predictive model for developing POI risk at different ages in female cancer survivors. [ref]</p>
<p>Genetic Risk for Primary Ovarian Insufficiency</p>
<p>Aside from clinical risk factors, genetic studies have shown that POI  is a complex, heterogeneous disorder that is influenced by multiple genetic components<a href="https://www.zotero.org/google-docs/?FpiE89">[16]</a>. Genetic studies have shown that the genetic variations on the X chromosome contribute mostly to the etiology of POI. Meanwhile, increasing attention has focused on autosome single-gene variations known to regulate follicle development and maturation. For example, Rajkovic et al. <a href="https://www.zotero.org/google-docs/?badzZj">[17]</a> discovered that the Newborn ovary homeobox gene (NOBOX) plays a critical role in early folliculogenesis.<a href="https://www.zotero.org/google-docs/?Kq1bPI">[18]</a> Deficiency in NOBOX disrupts early folliculogenesis and oocyte-specific gene expression, accelerating post-natal oocyte loss and abolishing the transition from primordial to growing follicles. Moreover, The mutations in follicle-stimulating hormone receptor genes, such as FSHR and LHCGR, were found to affect the development and maturation of follicles and oocytes [ref]. So far, more than sixty genetic variants have been identified, accounting for 10–13% of the variance of POI in the general population<a href="https://www.zotero.org/google-docs/?DRSj1Y">[19]</a>.  A list of several genes that affect the development and maturation of POI is included in the appendix </p>
<p>Genetic variants and Genome-wide Association Study (GWAS)</p>
<p>The DNA sequence is formed from a chain of four nucleotide bases: A, C, G, and T.  All humans have near-identical DNA sequences across the estimated six billion-letter codes for their genome.</p>
<p><a href="https://www.zotero.org/google-docs/?GDWxvk">[20]</a> However, slight differences exist between individuals.  If more than 1% of a population does not carry the same nucleotide at a specific position within the DNA sequence, then this variation can be classified as a single nucleotide polymorphism (SNP) or genetic variant. <a href="https://www.zotero.org/google-docs/?0P7REo">[21]</a></p>
<p>Many techniques were applied to discover the genetic variants for different phenotypes. A popular technique, Genome-wide Association  Study (GWAS), has been widely conducted to study the genetics of phenotypes in recent decades<a href="https://www.zotero.org/google-docs/?JFvUz3">[22]</a>, which aims to find common variants<a href="https://www.zotero.org/google-docs/?PZrbTd">[23]</a>. GWAS technique consists of screening and comparing genetic variants in patients with the disease of interest ( or phenotype) and healthy controls. In GWAS, the association between a genetic variant to the phenotype is observed when there is a statistically significant difference of the genetic variant between diseased patients and healthy controls. </p>
<p>GWAS studies have been conducted in several ancestries. As of 2019, 78% of the GWAS studies were performed in European ancestry, 10% were conducted in Asian ancestry. For the POI trait, There have been five GWAS studies so far. Each of the five studies had a small sample size, with the largest being roughly 1300 people. Four of them were performed among people of European or East Asian ancestry in the general population, and the remaining one was conducted in the childhood cancer survivors. A summary of the studies was provided in the appendix.[attach the table from catlog]. More GWASs studies with a larger study sample size were available for age at natural menopause. There were 14 GWAS studies conducted, the biggest of which included 69626 people of European ancestry.</p>
<p>GWAS data for Age at Natural Menopause in the General Population</p>
<p>Ideally, the GWAS information of POI is the perfect data source for studying the association between common variants and POI.  However, the existing GWAS data for POI in the general population was limited by the small sample size of the GWAS studies. Instead, this study focused on the GWAS data of age at natural menopause in the general population, as it has been hypothesized that age at natural menopause and POI are possibly manifestations of the same underlying genetic susceptibility owing to the inheritance patterns observed.<a href="https://www.zotero.org/google-docs/?yjmpd8">[24]</a>  Studies also showed that POI and age at natural menopause share common genetic factors involved in DNA repair and maintenance [ref]. The following describes some main findings of the two most recent GWAS studies with a reasonable sample size for age at natural menopause in the general population.</p>
<p>A meta-analysis of GWAS conducted in the general population analyzed up to 69,360 women of European ancestry.<a href="https://www.zotero.org/google-docs/?WIkCPi">[25]</a> The study showed that 1,208 out of a total of ~2.6 million genetic variants reached the genome-wide significance threshold (P &lt; 5 × 10−8) for association with age at natural menopause. Among those significant genetic variants, 54 independent genetic variants at 44 genetic loci were associated with age at menopause, explaining 6% of the variance in age at natural menopause.  The variance explained increased to 21% for the top 29,958 independent variants with an association P value less than 0.05. A more recent study included 119,160 samples of European population from the UK-Biobank release 2, and examined about 9.5 million genetic variants, the study identified 74 independent genetic variants that were associated with POI.<a href="https://atlas.ctglab.nl/traitDB/3366" class="uri">https://atlas.ctglab.nl/traitDB/3366</a> </p>
<p>GWAS data for Primary Ovarian Insufficiency in the Cancer Survivors</p>
<p>The genetics of menopause in the cancer survivor population is less investigated. To date, only Brooke et al. have conducted a GWAS study in the cancer survivor population.[ref] The study conducted GWAS to identify genetic variants associated with clinically diagnosed POI in 799 female cancer survivors in the St. Jude Lifetime Cohort Study (SJLIFE). The analyses were adjusted for cyclophosphamide equivalent dose of alkylating agents and ovarian radiotherapy (RT) dose. The results showed that 20 out of 830 884 genotyped variants had a P value less than 10-5. Among them, 13 genetic variants (upstream of the Neuropeptide Receptor 2 gene) were seen in more than 50% of POI patients and were associated with POI prevalence.  A haplotype formed by 4 of the 13 variants showed association with POI for patients exposed to ovarian RT, indicating an interaction between genetics and radiation treatment. Replication was performed in 1624 survivors from the Childhood Cancer Survivor Study (CCSS), and the association was also observed between the haplotype and POI for patients exposed to ovarian radiotherapy (OR= 3.97, 95%CI=[1.67 to 9.41], P= .002).</p>
<p>Potential of Polygenic Risk Score</p>
<p>Although the increasing use of GWAS has led to the identification of many variants associated with POI,  the effect size of genetic variants identified from GWAS are typically small and account for only a small fraction of association, meaning that single variants have limited predictive power.<a href="https://www.zotero.org/google-docs/?fah5s9">[26]</a> Thus, this study evaluated the genetic risk for POI in the form of a polygenic risk score (PRS), or a score that combines the estimated effects of many disease-associated genetic variants reported in published GWAS data. PRSs have been proposed as a genetic risk prediction tool for a wide range of diseases [ref];  According to the PGS catalog, an open database of published polygenic risk scores, 829 polygenic risk scores were built over 214 traits as of  July 2021. A clinically useful PRS would allow clinicians to identify individuals at elevated risk of disease, thus informing disease screening, therapeutic interventions, and life planning to prevent or delay disease onset.</p>
<div id="methodology-review" class="section level2" number="1.1">
<h2><span class="header-section-number">1.1</span> <strong>Methodology Review</strong></h2>
<div id="polygenic-risk-score" class="section level3" number="1.1.1">
<h3><span class="header-section-number">1.1.1</span> Polygenic Risk Score</h3>
<p>A polygenic risk score or PRS (also termed a polygenic score or genetic risk score) estimates an individual’s genetic liability to a trait or disease.<a href="https://www.zotero.org/google-docs/?xSMcco">[26]</a> PRS is derived from genotype profile and relevant genome-wide association study (GWAS) data. PRS is a sum of genome-wide genotypes, weighted by corresponding genotype effect size estimates derived from GWAS summary statistics. The general steps for computing PRS include selecting base and target data, quality control of the base and target data, and finally, the construction of PRS. </p>
<p>Base and Target Data</p>
<p>Two key input data sets are required to characterize PRS based on the definition. The base data (GWAS studies), consisting of summary statistics of the association between genetic variants and phenotypes, are needed to characterize PRS. Target data, usually new samples independent of the base samples and with genotype and phenotype data for each individual, are used to perform the PRS analysis. </p>
<p>The base data (i.e., summary statistics of GWAS) are usually publicly available. the following summary statistic information is needed for the PRS construction: </p>
<table>
<colgroup>
<col width="24%" />
<col width="75%" />
</colgroup>
<thead>
<tr class="header">
<th>Summary Statistics</th>
<th>Description</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>Genetic variant identifiers</td>
<td>the coordinate of the genetic variant including chromosome and base pair position, human genome assembly/build</td>
</tr>
<tr class="even">
<td>Effect allele</td>
<td>the allele that was coded for association testing and can either increase or decrease risk.</td>
</tr>
<tr class="odd">
<td>Reference allele</td>
<td>the non-effect allele</td>
</tr>
<tr class="even">
<td>Effect allele frequency</td>
<td>The frequency of the effect allele of a genetic variant in the population</td>
</tr>
<tr class="odd">
<td>Regression coefficient/effect size</td>
<td><p>the change in the trait value or disease risk with each additional copy of the effect allele.</p>
<p>(i.e., the coefficients of regression)</p></td>
</tr>
<tr class="even">
<td>Standard error</td>
<td>Standard error of the regression coefficient</td>
</tr>
<tr class="odd">
<td>Sample size</td>
<td>The sample size of the base data</td>
</tr>
<tr class="even">
<td>P-value</td>
<td>The significance of the genetic variant</td>
</tr>
</tbody>
</table>
<p>Selection of  Base Data (GWAS)</p>
<p>When it comes to the selection of GWAS studies, several factors were taken into account, including the sample size of the GWAS, the ancestry of the samples, the phenotypes being studied in the GWAS, and the heritability of the variants.</p>
<p>Sample Size of GWAS: Given that the PRS is built based on the summary statistics from GWAS studies, the GWAS studies should have a reasonable sample size so that it is powerful enough to detect the association between genetic variants and phenotype and ultimately increase the predictive ability of the PRS. GWAS studies with a sample size larger than 10,000 are recommended. </p>
<p>Ancestry of the Sample: The genetic profiles of different ancestry groups are significantly different, as the allele frequencies and the correlation between genetic variants vary among different ancestry groups. The estimated effect size of a given genetic variant might be different from population to population. Therefore, the PRS generated from GWAS summary statistics using target samples is usually limited to a specific population. For this reason, the ancestry of the base and target samples should be the same to ensure an accurate estimation.  </p>
<p>Phenotype Being Studied: The phenotype studied in the base data should be the same as the phenotype of interest in the PRS construction. However, some phenotypes might share similar genetic architecture. GWAS studies for other traits that are relevant to the phenotype of interest could be a substitute when GWAS studies for phenotypes of interest are not available. </p>
<p>Heritability:  The heritability measures the proportion of variations in a trait that can be attributed to genetics. A heritability estimate larger than 0.05 is recommended when performing PRS analyses to ensure the predictive power of PRSs. The heritability estimate is usually reported in the GWAS studies. </p>
<p>As discussed in the above session, the inclusion of the GWAS in this study thus can be summarized as follows:</p>
<ul>
<li><p>GWAS conducted in general population or cancer survivor sample(s) of predominantly European ancestry</p></li>
<li><p>Recommended sample size ≥10,000</p></li>
<li><p>Phenotype definition in GWAS/meta-analyses is relevant for the study of menopause-related phenotypes in survivors. That is, age at natural menopause or POI.</p></li>
<li><p>GWAS studies with complete summary statistics data</p></li>
<li><p>GWAS studies conducted with appropriate standard sample/variant quality control procedures</p></li>
</ul>
<p>Quality Control of Base and Target data</p>
<p>Following the base and target data selection is the quality control, mainly including standardizing the genome build and removing overlapping samples between the base and target data.</p>
<p>Standardize the Genome Build: Currently, there are several different genome builds.<a href="https://www.zotero.org/google-docs/?h6ZcK7">[27]</a> Thus the chromosomal coordinates (genomic positions) for a particular gene in one genome build may not be the same as those in the next release of the build unless the gene happens to lie on a completely sequenced and unrevised chromosome.<a href="https://www.zotero.org/google-docs/?dfStya">[28]</a> Therefore, if the positions of the genetic variants were assigned on different genome builds, position converting should be done to achieve the correspondence of the positions of the genetic variants between the base and target data.<a href="https://www.zotero.org/google-docs/?GEaWFS">[29]</a> We can migrate the positions of the genetic variants of the base data to that of the target data, or vice versa. Some online tools such as Ensembl <a href="https://www.zotero.org/google-docs/?hpRJF9">[30]</a> or LiftOver <a href="https://www.zotero.org/google-docs/?SoLtIL">[31]</a> can be used to standardize the genome build across base and target data.</p>
<p>Sample Overlapping: Study samples that overlap between the base and target data must be removed as sample overlapping can result in inflation of the association between the PRS and the phenotype tested in the target data. <a href="https://www.zotero.org/google-docs/?q5Z5bZ">[32]</a> A study showed that the level of inflation is proportional to the fraction of the target sample that overlaps the base sample.</p>
<p>Calculating Polygenic Risk Score</p>
<p>Following the quality control of both base and target samples, the next step is to calculate the PRS for individuals in the target data. As aforementioned, PRS is usually a weighted sum of the number of risk alleles and thus can be given by:</p>
<p>PRS=iixi</p>
<p>Where i is an indicator for the i th genetic variant, i is the log odds ratio or the coefficient of a linear or logistic regression.</p>
<p>Clumping and Thresholding </p>
<p>The clumping and thresholding (C+T) method is a traditional way used to derive PRS where the clumping is used to deal with the correlation between genetic variants, and the thresholding is used for the selection of genetic variants and the adjustment of the effect sizes. </p>
<p>The correlation (term as linkage disequilibrium in bioinformatics) between genetic variants from GWAS is an important factor that should be considered. Regression analysis accounting for known covariates is typically used in GWASs to study the association between genetic variants and the phenotype of interest, which means the association was tested one genetic variant at a time. Therefore, the genetic variants may be correlated with one another.<a href="https://www.zotero.org/google-docs/?tIPUnj">[33]</a></p>
<p>To account for the correlation, we need to calculate the correlation between genetic variants. As the genotyping data of the base data is usually unavailable, reference sample data of the same ancestry with the base data could be used to estimate the correlation. A widely used reference sample source is the 1000 Genomes Project data. <a href="https://www.zotero.org/google-docs/?Rdj2Eh">[34]</a> An alternative is to use the target data as the reference sample for estimating the correlation if the sample size of the target data is more than 1000. However, if base and target samples are drawn from different populations, the base data correlation may be poorly approximated, and the accuracy of PRS is reduced accordingly.</p>
<p>Clumping circles through genetic variants iteratively. It computes the pairwise correlation ( R2) between the current genetic variant and the nearby variants (for example, within 250 kilobases of the current genetic variant) using the reference samples’ genotyping data. If the correlation coefficient is beyond a pre-specified threshold (e.g., R2=0.1, 0.4, etc.), the genetic variant with a larger P-value (less significant) is removed from the GWAS summary statistics . </p>
<p>Given that not all genetic variants in the GWAS studies are associated with the phenotype of interest after clumping, another key factor we need to consider in the construction of PRS is the inclusion and exclusion of genetic variants. In the C+T method, the inclusion of genetic variants is determined by the GWAS association P-value. Thus this step is termed thresholding. Specifically, the thresholding only allows genetic variants to be included in the construction of PRS if the GWAS association P-value is under a certain threshold (e.g., P&lt;10-5, the critical value in GWAS studies is more stringent to provide confidence that the association is true <a href="https://www.zotero.org/google-docs/?0OEZ2e">[35]</a>). This process can be treated as an efficient way to adjust the effect size of the genetic variants, where all the genetic variants with P-value larger than the threshold are adjusted to an effect size estimate of zero, and perform no adjustment on the effect size estimates of those genetic variants included (i.e., variants with P values less than the threshold).</p>
<p>In the C+T method, researchers can specify the pairwise correlation coefficient R2for clumping and different p-value thresholds to generate multiple candidate PRSs. Another intuitive way to compute a PRS is to utilize the independent genetic variants with the most significant P values reported in the GWAS when the target data is unavailable.</p>
</div>
<div id="risk-prediction" class="section level3" number="1.1.2">
<h3><span class="header-section-number">1.1.2</span> Risk Prediction</h3>
<p>Time-specific Logistic Regression </p>
<p>Logistic regression is one of the most commonly used approaches for modeling the relationship between variables and a binary outcome. The general form of logistic regression can be expressed as:</p>
<p>𝑙𝑛(𝑝/(1 − 𝑝)) = 𝛽0 + 𝛽1 𝑋1 + 𝛽2 𝑋2. . . +𝛽𝑘𝑋𝑘</p>
<p> where p denotes the probability of the outcome of interest occurring, and p/(1-p) represents the odds of the outcome. 𝑙𝑛 denotes the natural logarithm. 𝑋1, 𝑋2, … 𝑋𝑘 are the predictors (could be potential interactions) and 𝛽1, 𝛽2, …, 𝛽𝑘 are their coefficients, and 𝛽0 is the intercept. Logistic regression models the log odds of an outcome using a linear combination of predictors. Several assumptions are required to use the logistic regression, including a linear relationship between the covariates and the log odds of the outcome, no multicollinearity among the covariates, and independence among individuals.</p>
<p>Logistic regression explores the risk factors for the outcome and determines the relative importance of risk factors with respect to the outcome. The odds ratio, given by exp(), is used to measure the association. For a continuous variable, the odds ratio can be interpreted as holding all other covariates constant, the odds of the outcome of interest occurring will increase by exp() for one unit increase in the covariate value. For a categorical variable, the odds ratio can be interpreted as: the odds of the outcome of interest occurring in a specific category is exp() times the odds of that in the reference category. The absolute risk of the outcome occurring can be estimated in the logistic regression, which is represented by the probability of the occurrence of the outcome. </p>
<p>Logistic regression is very popular for its interpretability. However, logistic regression cannot handle time-to-event data, as logistic regression does not take the time of the event’s occurrence into consideration in the modeling. In this study, POI is determined by two elements: the ovarian status and the age at menopause (i.e., time-to-event data). Thus censoring is a concern when modeling POI risk. For example, a survivor’s POI status was censored if she was 27 at her last follow-up and had a normal menstrual function. She was still at risk of POI. However, the actual outcome is not observable due to censoring. </p>
<p>The inverse-probability-of-censoring weighting (IPCW) method was employed in a previous study to account for the censored observations. Briefly, the IPCW weights were obtained by modeling the censoring process using the same set of covariates, such as age at diagnosis and radiation dosage to the ovary, for modeling the POI status. Censored individuals will thus contribute to the risk model through the IPCW weights. Individuals with known POI status will be given weights in the estimation of the logistic regression model. Therefore, we call this model “time-specific logistic regression”.</p>
<p>Apart from censoring, the competing risk event of surgical premature menopause (had bilateral oophorectomy) needs to be considered. Female childhood cancer survivors who had bilateral oophorectomy before the age of 40 would not develop menopause naturally (i.e., are no longer at risk of natural menopause). The competing risk is considered in the IPCW method, where there is an indicator variable for the event (menopause, menstruation, or surgical premature menopause)</p>
<p>Time-specific logistic regression is used to estimate the POI risk in this study. As section ## mentioned, clinical risks such as radiation therapy and genetic variation contribute to the POI risk. Thus both clinical and genetic components should be included in the logistic regression. However, instead of estimating the effect for each clinical risk factor, a clinical risk score or linear predictor –the combination of regression coefficients and the predictors from the existing clinical model for POI risk – can be included in the model to represent the clinical risk.  </p>
<p>To get a genuine estimate of the clinical risk, the effect of the clinical risk score can be further adjusted by remodeling the relationship between the clinical risk score and the outcome of interest on new data. The remodeling can be written as: </p>
<p>f(ynew)=a+b*linear predictor</p>
<p>The linear predictor is the clinical risk score estimated from the clinical model. The coefficient of the clinical risk score (b) represents the amount of adjustment required to obtain a “true” effect of the clinical factors on POI. This remodeling process is called recalibration. and b is named the calibration slope. Ideally, b will has a value of 1, representing the best fit on the data. A calibration slope less than 1 indicates the amount we need to reduce on effects of predictors on average to make the model well-calibrated for new patients from the underlying population. In contrast, a calibration slope larger than 1 indicates the amount needed to increase for better calibration. Finally, the adjusted clinical risk score, i.e., b*linear predictor, is included in the logistic regression as an offset where a coefficient estimate of one was assigned in the modeling.</p>
<p>The PRS then is added to the model to account for some of the variations of POI risk among individuals that could not be explained by the clinical risk. Potential interactions between clinical risk and PRS can be examined by including the interaction. Finally, we can compare the clinical risk model adjusted by the calibration slope with the PRS+clinical risk score models. This allows us to see the incremental value in the prediction of incorporating PRS in the clinical model.</p>
</div>
<div id="evaluation-of-model-performance" class="section level3" number="1.1.3">
<h3><span class="header-section-number">1.1.3</span> Evaluation of Model Performance</h3>
<p>We need to evaluate the quality of the predictive model after model development. For the prediction of a binary outcome, we typically examine three different perspectives: </p>
<ol style="list-style-type: decimal">
<li><p>the closeness of the predictions are to the observed outcome (overall performance);</p></li>
<li><p>the discrimination power of the prediction model;</p></li>
<li><p>and the calibration of the prediction model</p></li>
</ol>
<p>To answer these questions,  the following performance measures are used and will be calculated under a five-fold cross-validation framework.</p>
<div id="the-overall-performance-scaled-brier-score" class="section level4" number="1.1.3.1">
<h4><span class="header-section-number">1.1.3.1</span> The Overall Performance: Scaled Brier Score</h4>
<p>The overall performance is quantified by the distance between the predicted outcome (for a binary outcome, it refers to the predicted probability) and the observed outcome. The squared difference, defined as the Brier score, is usually calculated to quantify the distance. The formula of Brier score is calculated as the mean squared error of the prediction, written as: </p>
<p>BS =1Nt=1N(pt-yt)2</p>
<p>Where pt is the probability of the t th individual, N is the number of individuals being predicted, yt is the actual observed outcome of the t th individual. Brier score corresponds to the goodness of fit of the model, and it takes values in the interval [0,1]. A smaller distance between the predicted and observed outcome indicates a better prediction. </p>
<p>However, the range of Brier scores changes by incidence level. For instance, with incidence = .5, the Brier score will range from 0 for a perfect prediction to a maximum of .25 for a non-informative model where the overall proportion of the event of interest is usually given as the predicted probability of the event occurring for each individual in the dataset. While with incidence decreased from .5 to .1, the maximum Brier score will decrease to 0.09. Thus, the interpretation and comparison of multiple Brier scores is challenging when Brier scores are calculated from different incidences. </p>
<p>One alternative way to evaluate the overall performance is to use scaled Brier score (sBrS).<a href="https://www.zotero.org/google-docs/?AJ2oWi">[36]</a> It is calculated as:</p>
<p>sBrS = 1-BSBSref=(BSref-BSBSref)</p>
<p>where BSref is the Brier score of the baseline prediction model(s) which need to be improved. The baseline prediction models could be any pre-existing models. More commonly, a naive model is employed as a baseline model, in which each individual in the data set is given a constant predicted probability, which is the overall frequency of the outcome of interest. The Brier score  for a naive model is calculated as:</p>
<p>BSref=1Nt=1N(y-yt)2</p>
<p>Where y is the event rate of the outcome of interest of the samples. As we can see from the expression, BSS is very similar to the coefficient of determination (R2) in regression.<a href="https://www.zotero.org/google-docs/?oqVUS7">[37]</a> BSS measures the fractional amount of improvement in the Brier score of a model compared to the baseline model and is more interpretable than the Brier score. A BBS of zero indicates that the prediction is only as good as the baseline model, and a BSS of one suggests that the prediction is perfect. A BBS less than zero indicates that the prediction is even worse than that of the baseline model.</p>
</div>
<div id="the-discrimination" class="section level4" number="1.1.3.2">
<h4><span class="header-section-number">1.1.3.2</span> The  Discrimination</h4>
<p>Discrimination assesses a predictive model’s ability to discriminate those who have the outcome from those who do not, i.e., the model’s ability to classify the outcome as event or non-event accurately. The area under the receiver operating characteristic (ROC) curve is the most commonly used metric to evaluate the discrimination of prediction algorithms. The ROC curve plots true-positive rate (TPR, or sensitivity) versus false-positive rate (FPR, or 1-specificity) of the predictive models over all possible cutoffs for the probability of an outcome, where true positive rate and false positive rate were calculated as:</p>
<p>The area under the ROC curve (AUC) summarizes the model performance over all possible cutoffs. AUC represents the probability that a randomly chosen observation with a positive outcome (case) has a higher predicted risk score than a randomly chosen observation with a negative outcome (noncase). According to the definition of AUC, it can be calculated by randomly selecting a case and a non-case as a pair, then calculating the proportion of pairs where the case has a higher predicted risk score than that of the non-case over all possible pairs. </p>
<p>AUC ranges from 0.5 to 1.0. An AUC of 0.5 (A ROC curve with the line at 45 degrees ) represents the true positive rate equals the false positive rate, indicating that the predictive model is unable to discriminate between positive and negative outcomes. An AUC value greater than 0.5 implies that the prediction model has some ability to distinguish between positive and negative outcomes. The higher the AUC, the better the discrimination performance of the predictive algorithm. A perfect predictive model has an AUC of 1.0. </p>
</div>
<div id="the-calibration" class="section level4" number="1.1.3.3">
<h4><span class="header-section-number">1.1.3.3</span> The Calibration</h4>
<p>The calibration is another key property of predictive models. The calibration corresponds to the agreement between the observed outcomes and predicted values, and evaluates the reliability of the predictions. A well-calibrated model is that for individuals with a predicted risk of p%, the observed frequency of events among those individuals should be approximately p%.<a href="https://www.zotero.org/google-docs/?CkBl16">[38]</a> For example, if the mean predicted probability for a group of individuals is 20%, the model is well-calibrated if the observed probability is close to 20%. </p>
<p>Spiegelhalter-z Statistics</p>
<p>The Brier score is an overall measure of performance and can be decomposed to calibration and discrimination. Spiegelhalter-z statistics is used to measure the calibration aspect of the Brier score. By expanding the square in equation ## (on page 22), the Brier score can be decomposed into:</p>
<p>BS =1Ni=1N(yi-pi)2</p>
<p>                                               =1Ni=1N(yi-pi)(1-2pi)+1Ni=1Npi(1-pi)</p>
<p>where the first term measures lack of calibration and the second term measures lack of discrimination of the predictions. The Spiegelhalter-z statistics is defined as:  </p>
<p>z=i=1n(yi-pi)(1-2pi)i=1n(1-2pi)2pi(1-pi)</p>
<p>It asymptotically follows a standard normal distribution. The Spiegelhalter-z statistics has a value of zero under the null hypothesis of perfectly calibrated, statistically significant scores.</p>
<p>(i.e., z &lt; −1.96 or z &gt; 1.96) generally indicate poor calibration. </p>
<p>Calibration Plot</p>
<p>The plot helps visualize the calibration. The calibration plot usually has the prediction on the x-axis and the outcome on the y-axis. The perfect predictions should be at the 45-degree line and help for orientation. For binary outcomes, we can first group individuals by similar predicted probabilities, e.g., group by quantiles. Then we compare the mean observed proportions with the event of each subgroup to its mean predicted probabilities. Moreover, smoothing techniques such as loess algorithm can be applied to estimate the relationship between the observed proportions of the event and the predicted probabilities. </p>
</div>
</div>
</div>
<div id="the-ccss-original-cohort" class="section level2" number="1.2">
<h2><span class="header-section-number">1.2</span> <strong>The CCSS Original Cohort</strong></h2>
<p>As mentioned in section ##, the survival rates of childhood cancers have risen steadily over the past several decades.<a href="https://www.zotero.org/google-docs/?Bwad57">[39]</a> Long-term survivors are at risk for a variety of adverse effects, including early death, decreased fertility, and a lower quality of life overall. This has increased the need for assessing the long-term morbidity and mortality, as well as the side effects of cancer treatments.</p>
<p>The Childhood Cancer Survivor Study (CCSS), a multi-institutional retrospective cohort study, was established to assess the late mortality, subsequent neoplasms, adverse cardiac and pulmonary outcomes, fertility and offspring, family history of cancer and health-related behaviors of cancer survivors. The CCSS original cohort was first established in 1994 and recruited survivors in North America that met the following eligibility criteria: (1) diagnosed between January 1st 1970 and December 31st 1986 ; (2) age less than 21 at diagnosis; (3) lived for at least five years after the date of a cancer diagnosis. </p>
<p>Baseline Demographic Data</p>
<p>Following the establishment and recruitment procedure, a baseline questionnaire was completed by 14,054 eligible subjects in CCSS original cohort. The questionnaire consisted of 289 questions regarding demographics, medical care practices, prescription medications, medical conditions and so on.  Five follow-up surveys were released to obtain the updated information regarding the health conditions and monitor the potential adverse effects. Demographic information was requested from CCSS. Of particular interest to this study are age, sex, race, cancer type,  age at cancer diagnosis.<a href="https://www.zotero.org/google-docs/?9yIX6B">[40]</a></p>
<p>Treatment Data</p>
<p>All participants who completed the baseline questionnaire were asked to sign a consent form to allow access to all medical records since their cancer diagnosis. Then a detailed summary of the cancer treatment information including chemotherapy, radiation therapy and surgery for each cohort member was extracted from the medical record. Information was collected regarding the specific chemotherapy agents and their respective doses, as well as radiation dose and location.</p>
<p>Genotyping Data</p>
<p>The buccal cell genomic DNA information of eligible participants who had completed the baseline questionnaire was also collected during May 1999 and June 2006.[ref] A specimen collection kit was mailed to participants, which included a cover letter outlining the study, a consent form, an instruction sheet, a 45 mL bottle of mouthwash, a specimen collection container, return mail labels, and postage. The participants were asked to return the sample to the Molecular Genetic Laboratory in Cincinnati, Ohio. A total number of 5739 participants have genotype data available, among which 2958 are female participants.<a href="https://www.zotero.org/google-docs/?ssis8u">[41]</a> The genotyping data was requested from CCSS and standard quality control (QC) [QC should be listed in appendix] of CCSS original cohort genotype data was performed. <a href="https://www.zotero.org/google-docs/?fIYsIX">[42]</a></p>
<p>Definition of POI Outcomes</p>
<p>The outcome of interest in this study is POI, defined as either: (1) experiencing menopause naturally before the age of 40 years for individuals who had menarche before cancer treatment, or (2) never experiencing menarche by the age of 18 years for individuals who did not have menarche before cancer treatment. [ref]</p>
<p>Two variables are associated with the outcomes: The ovarian status and the age at the event onset. The above-established definition and patients’ self-reported menstrual history information in the baseline and follow-up 1, 4 and 5 questionnaires help classify patients as POI, surgical premature menopause (SPM, had bilateral oophorectomy) or normal.  Ambiguous cases whose ovarian status could not be determined were further manually reviewed by endocrinologists, based on the patients’ responses for menstrual history questions; Age at event onset is also derived from the CCSS surveys. The age at last menstrual period or surgical time informs age at menopause or surgical ovarian removal, respectively. For patients with incomplete age information, age at event onset were imputed in consultation with endocrinologists [ref] (Appendix: age assignment algorithm) A copy of the survey questions specific to determining ovarian status is included in the appendix.</p>
<p>CCSS Study Sample Eligibility</p>
<p>Data of the female cancer survivors in the CCSS original cohort study was obtained in order to develop risk predictive models for POI in female childhood cancer survivors. The inclusion and exclusion criteria were well claimed and this process has been done in previous studies.[ref] The detailed exclusion criteria is included in the <strong>appendix[ref]</strong> Furthermore,participants without Genome-wide Association Study (GWAS) data and who were not of European descent were excluded. Individuals who overlapped with the St Jude Lifetime Cohort Study (SJLIFE) were also excluded.</p>
<p>Of the 4541 cancer survivors received from CCSS original cohort,  4432 female survivors remained in the study sample after applying the exclusion criteria established in previous studies. Additionally, 1977 individuals were excluded for missing Genome-wide Association Study (GWAS) data , and 250 individuals who were not of European descent were excluded. The total study sample consisted of 1985 individuals, with %  POI cases. A description of the data has been done in a previous study [ref], Thus the detailed description of data is not provided in this paper.  However, a brief description of the data is provided in the appendix.</p>
</div>
<div id="references" class="section level2" number="1.3">
<h2><span class="header-section-number">1.3</span> <strong>References</strong></h2>
<p><a href="https://www.zotero.org/google-docs/?yU4ZoA">1. Lund LW, Schmiegelow K, Rechnitzer C, Johansen C. A systematic review of studies on psychosocial late effects of childhood cancer: structures of society and methodological pitfalls may challenge the conclusions. Pediatr Blood Cancer. 2011;56:532–43.</a></p>
<p><a href="https://www.zotero.org/google-docs/?yU4ZoA">2. Canada PHA of. Cancer in Children in Canada (0-14 years). 2012. &lt;https://www.canada.ca/en/public-health/services/chronic-diseases/cancer/cancer-children-canada-0-14-years.html.&gt; Accessed 5 Aug 2021.</a></p>
<p><a href="https://www.zotero.org/google-docs/?yU4ZoA">3. Stats - Kids Cancer Care. &lt;https://www.kidscancercare.ab.ca/childhood-cancer/stats.&gt; Accessed 5 Aug 2021.</a></p>
<p><a href="https://www.zotero.org/google-docs/?yU4ZoA">4. Marwick C. Childhood cancer survivors experience long term side effects. BMJ. 2003;327:522.</a></p>
<p><a href="https://www.zotero.org/google-docs/?yU4ZoA">5. Reproductive health in female survivors of childhood cancer - Gelson - 2016 - The Obstetrician &amp; Gynaecologist - Wiley Online Library. &lt;https://obgyn.onlinelibrary.wiley.com/doi/10.1111/tog.12338.&gt; Accessed 5 Aug 2021.</a></p>
<p><a href="https://www.zotero.org/google-docs/?yU4ZoA">6. Levine JM, Whitton JA, Ginsberg JP, Green DM, Leisenring WM, Stovall M, et al. Nonsurgical premature menopause and reproductive implications in survivors of childhood cancer: A report from the Childhood Cancer Survivor Study. Cancer. 2018;124:1044–52.</a></p>
<p><a href="https://www.zotero.org/google-docs/?yU4ZoA">7. Chemaitilly W, Li Z, Krasin MJ, Brooke RJ, Wilson CL, Green DM, et al. Premature Ovarian Insufficiency in Childhood Cancer Survivors: A Report From the St. Jude Lifetime Cohort. J Clin Endocrinol Metab. 2017;102:2242–50.</a></p>
<p><a href="https://www.zotero.org/google-docs/?yU4ZoA">8. Rudnicka E, Kruszewska J, Klicka K, Kowalczyk J, Grymowicz M, Skórska J, et al. Premature ovarian insufficiency – aetiopathology, epidemiology, and diagnostic evaluation. Prz Menopauzalny. 2018;17:105–8.</a></p>
<p><a href="https://www.zotero.org/google-docs/?yU4ZoA">9. Sklar CA, Mertens AC, Mitby P, Whitton J, Stovall M, Kasper C, et al. Premature menopause in survivors of childhood cancer: a report from the childhood cancer survivor study. J Natl Cancer Inst. 2006;98:890–6.</a></p>
<p><a href="https://www.zotero.org/google-docs/?yU4ZoA">10. Miro F, Parker SW, Aspinall LJ, Coley J, Perry PW, Ellis JE. Sequential classification of endocrine stages during reproductive aging in women: the FREEDOM study. Menopause. 2005;12:281–90.</a></p>
<p><a href="https://www.zotero.org/google-docs/?yU4ZoA">11. Gold EB, Bromberger J, Crawford S, Samuels S, Greendale GA, Harlow SD, et al. Factors Associated with Age at Natural Menopause in a Multiethnic Sample of Midlife Women. American Journal of Epidemiology. 2001;153:865–74.</a></p>
<p><a href="https://www.zotero.org/google-docs/?yU4ZoA">12. prevalence and phenotypic characteristics of spontaneous premature ovarian failure: a general population registry-based study | Human Reproduction | Oxford Academic. &lt;https://academic.oup.com/humrep/article/30/5/1229/589165?login=true.&gt; Accessed 5 Aug 2021.</a></p>
<p><a href="https://www.zotero.org/google-docs/?yU4ZoA">13. Younis JS. Ovarian aging and implications for fertility female health. Minerva Endocrinol. 2012;37:41–57.</a></p>
<p><a href="https://www.zotero.org/google-docs/?yU4ZoA">14. Shah D, Nagarajan N. Premature menopause – Meeting the needs. Post Reprod Health. 2014;20:62–8.</a></p>
<p><a href="https://www.zotero.org/google-docs/?yU4ZoA">15. Role of estrogen in diastolic dysfunction | American Journal of Physiology-Heart and Circulatory Physiology. &lt;https://journals.physiology.org/doi/full/10.1152/ajpheart.00859.2013.&gt; Accessed 7 Aug 2021.</a></p>
<p><a href="https://www.zotero.org/google-docs/?yU4ZoA">16. Pu D, Xing Y, Gao Y, Gu L, Wu J. Gene variation and premature ovarian failure: a meta-analysis. Eur J Obstet Gynecol Reprod Biol. 2014;182:226–37.</a></p>
<p><a href="https://www.zotero.org/google-docs/?yU4ZoA">17. Qin Y, Jiao X, Simpson JL, Chen Z-J. Genetics of primary ovarian insufficiency: new developments and opportunities. Human Reproduction Update. 2015;21:787–808.</a></p>
<p><a href="https://www.zotero.org/google-docs/?yU4ZoA">18. van Dooren MF, Bertoli-Avella AM, Oldenburg RA. Premature ovarian failure and gene polymorphisms. Current Opinion in Obstetrics and Gynecology. 2009;21:313–7.</a></p>
<p><a href="https://www.zotero.org/google-docs/?yU4ZoA">19. Goswami D, Conway GS. Premature ovarian failure. Horm Res. 2007;68:196–202.</a></p>
<p><a href="https://www.zotero.org/google-docs/?yU4ZoA">20. Polygenic Risk Scores. Genome.gov. &lt;https://www.genome.gov/Health/Genomics-and-Medicine/Polygenic-risk-scores.&gt; Accessed 5 Aug 2021.</a></p>
<p><a href="https://www.zotero.org/google-docs/?yU4ZoA">21. single nucleotide polymorphism / SNP | Learn Science at Scitable. &lt;http://www.nature.com/scitable/definition/single-nucleotide-polymorphism-snp-295.&gt; Accessed 7 Aug 2021.</a></p>
<p><a href="https://www.zotero.org/google-docs/?yU4ZoA">22. Perry JRB, Corre T, Esko T, Chasman DI, Fischer K, Franceschini N, et al. A genome-wide association study of early menopause and the combined impact of identified variants. Hum Mol Genet. 2013;22:1465–72.</a></p>
<p><a href="https://www.zotero.org/google-docs/?yU4ZoA">23. Christin-Maitre S, Tachdjian G. Genome-wide association study and premature ovarian failure. Ann Endocrinol (Paris). 2010;71:218–21.</a></p>
<p><a href="https://www.zotero.org/google-docs/?yU4ZoA">24. Rossetti R, Ferrari I, Bonomi M, Persani L. Genetics of primary ovarian insufficiency. Clin Genet. 2017;91:183–98.</a></p>
<p><a href="https://www.zotero.org/google-docs/?yU4ZoA">25. Day FR, Ruth KS, Thompson DJ, Lunetta KL, Pervjakova N, Chasman DI, et al. Large-scale genomic analyses link reproductive aging to hypothalamic signaling, breast cancer susceptibility and BRCA1-mediated DNA repair. Nat Genet. 2015;47:1294–303.</a></p>
<p><a href="https://www.zotero.org/google-docs/?yU4ZoA">26. Choi SW, Mak TS-H, O’Reilly PF. Tutorial: a guide to performing polygenic risk score analyses. Nat Protoc. 2020;15:2759–72.</a></p>
<p><a href="https://www.zotero.org/google-docs/?yU4ZoA">27. Genome Browser FAQ. &lt;https://genome.ucsc.edu/FAQ/FAQreleases.html#release1.&gt; Accessed 13 Aug 2021.</a></p>
<p><a href="https://www.zotero.org/google-docs/?yU4ZoA">28. Hinrichs AS, Karolchik D, Baertsch R, Barber GP, Bejerano G, Clawson H, et al. The UCSC Genome Browser Database: update 2006. Nucleic Acids Res. 2006;34 Database issue:D590-598.</a></p>
<p><a href="https://www.zotero.org/google-docs/?yU4ZoA">29. Information NC for B, Pike USNL of M 8600 R, MD B, Usa 20894. Data Changes that Occur Between Builds. National Center for Biotechnology Information (US); 2005. &lt;https://www.ncbi.nlm.nih.gov/books/NBK44467/.&gt; Accessed 13 Aug 2021.</a></p>
<p><a href="https://www.zotero.org/google-docs/?yU4ZoA">30. Ensembl genome browser 104. &lt;https://uswest.ensembl.org/index.html.&gt; Accessed 13 Aug 2021.</a></p>
<p><a href="https://www.zotero.org/google-docs/?yU4ZoA">31. Lift Genome Annotations. &lt;http://genome.ucsc.edu/cgi-bin/hgLiftOver.&gt; Accessed 13 Aug 2021.</a></p>
<p><a href="https://www.zotero.org/google-docs/?yU4ZoA">32. Wray NR, Yang J, Hayes BJ, Price AL, Goddard ME, Visscher PM. Pitfalls of predicting complex traits from SNPs. Nat Rev Genet. 2013;14:507–15.</a></p>
<p><a href="https://www.zotero.org/google-docs/?yU4ZoA">33. Marees AT, de Kluiver H, Stringer S, Vorspan F, Curis E, Marie‐Claire C, et al. A tutorial on conducting genome‐wide association studies: Quality control and statistical analysis. Int J Methods Psychiatr Res. 2018;27:e1608.</a></p>
<p><a href="https://www.zotero.org/google-docs/?yU4ZoA">34. 1000 Genomes | A Deep Catalog of Human Genetic Variation. &lt;https://www.internationalgenome.org/home.&gt; Accessed 13 Aug 2021.</a></p>
<p><a href="https://www.zotero.org/google-docs/?yU4ZoA">35. Panagiotou OA, Ioannidis JPA, for the Genome-Wide Significance Project. What should the genome-wide significance threshold be? Empirical replication of borderline genetic associations. International Journal of Epidemiology. 2012;41:273–86.</a></p>
<p><a href="https://www.zotero.org/google-docs/?yU4ZoA">36. Glahn HR, Jorgensen DL. CLIMATOLOGICAL ASPECTS OF THE BRIER P-SCORE. Monthly Weather Review. 1970;98:136–41.</a></p>
<p><a href="https://www.zotero.org/google-docs/?yU4ZoA">37. Hu B, Palta M, Shao J. Properties of R2 statistics for logistic regression. Statistics in Medicine. 2006;25:1383–95.</a></p>
<p><a href="https://www.zotero.org/google-docs/?yU4ZoA">38. Stevens RJ, Poppe KK. Validation of clinical prediction models: what does the “calibration slope” really measure? Journal of Clinical Epidemiology. 2020;118:93–9.</a></p>
<p><a href="https://www.zotero.org/google-docs/?yU4ZoA">39. Ries LAG. Cancer Incidence and Survival Among Children and Adolescents: United States SEER Program, 1975-1995. National Cancer Institute; 1999.</a></p>
<p><a href="https://www.zotero.org/google-docs/?yU4ZoA">40. progress-report-2013.pdf. &lt;https://ccss.stjude.org/content/dam/en_US/shared/ccss/documents/progress-report-2013.pdf.&gt; Accessed 7 Aug 2021.</a></p>
<p><a href="https://www.zotero.org/google-docs/?yU4ZoA">41. Public Access GWAS Data Tables. &lt;https://ccss.stjude.org/develop-a-study/gwas-data-resource/public-access-gwas-data-tables.html.&gt; Accessed 7 Aug 2021.</a></p>
<p><a href="https://www.zotero.org/google-docs/?yU4ZoA">42. Davies SM. CCSS Biospecimen  Repository. :8.</a></p>
<p><br />
</p>
<p><img src="https://lh6.googleusercontent.com/SCXuPVn7vScc2IKu6sBVb_cHRQKc7TWoENZ_Z97NlW8oHN_KzLq_xIF7d_v58EkkLtaHrs0zdixG8ekrRoiYe_11OcCdNjaZEW5DFt-jpxi7TLSLDY3naY7XrQ9PagwGYM6xC7tr=s0" /></p>
<p><br />
</p>

</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="index.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="polygenic-risk-score-construction.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"whatsapp": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": null,
"text": null
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": ["Lin_Thesis_Book.pdf", "Lin_Thesis_Book.epub"],
"search": {
"engine": "lunr",
"options": null
},
"toc": {
"collapse": "subsection"
}
});
});
</script>

</body>

</html>
